### PR TITLE
Add revision and get_latest for operator

### PR DESCRIPTION
**Pull request by shiyu22**

### CODE REVIEW

File: tests/unittests/runtime/test_user_pipeline.py [hunk 1]
```diff
@@ -406,3 +406,13 @@ def test_flatmap_coverage(self):
         )
         res = p(4).to_list()
         self.assertEqual(res, [[60], [40]])
+
+    def test_revision_latest(self):
+        p = (
+            Pipeline.input('a')
+            .map('a', 'b', ops.test_revision().revision('v1').get_latest())
```
Comment (junjiejiangjjj) [inline tests/unittests/runtime/test_user_pipeline.py:413]: .latest()  is enough

Comment (shiyu22) [reply tests/unittests/runtime/test_user_pipeline.py:413]: But it duplicated with the `latest` attr code:
```python
    @property
    def latest(self):
        return self._latest
```

Comment (junjiejiangjjj) [reply tests/unittests/runtime/test_user_pipeline.py:413]: > But it duplicated with the `latest` attr code:
> 
> ```python
>     @property
>     def latest(self):
>         return self._latest
> ```

is_latest ?

**Pull request merged**